### PR TITLE
RE2022-260: Add XY scatter plot endpoint to genome_attribs

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

**Comment by MrCreosote** [comment]
*&)*^&*( codecov

File: src/service/data_products/genome_attributes.py [hunk 1]
```diff
@@ -572,6 +572,83 @@ async def get_histogram(
...
 
+class XYScatter(BaseModel):
+    
+    xcolumn: Annotated[str, Field(
+        example="Completeness",
+        description="The name of the x column."
+    )]
+    ycolumn: Annotated[str, Field(
+        example="Contamination",
+        description="The name of the xy column."
```
Comment (Xiangs18) [inline src/service/data_products/genome_attributes.py:583]: "the name of the y column"

Comment (MrCreosote) [reply src/service/data_products/genome_attributes.py]: fixed

**Review by Xiangs18 — APPROVED** [review]
LGTM!

**Pull request merged**